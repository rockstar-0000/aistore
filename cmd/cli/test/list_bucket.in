ais archive gen-shards "ais://$BUCKET_1/tmp/shard-{0..4}.tar" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais ls ais://$BUCKET_1
ais ls ais://$BUCKET_1/
ais ls ais://$BUCKET_1
ais ls ais://$BUCKET_1 --props=name,size,version
ais ls ais://$BUCKET_1 --props=name,name,size,size,version
ais ls ais://$BUCKET_1 --props=all
ais ls ais://$BUCKET_1/

ais ls aws://$BUCKET_1/ // FAIL "cloud bucket"
ais ls az:// --all // FAIL "backend"
ais ls azure:// --all // FAIL "backend"

ais ls #ns // IGNORE
ais ls @ // IGNORE
ais ls @uuid --all // FAIL "remote cluster "uuid"

ais ls ais://#ns // IGNORE
ais ls ais://#ns/tmp // FAIL "bucket "ais://#ns/tmp" does not exist"
ais ls ais://@uuid#ns --all // FAIL "remote"
ais ls ais://@uuid#ns/tmp --all // FAIL "remote cluster "uuid" "

ais ls cloud://$BUCKET_1/ // FAIL "invalid backend provider "cloud""
ais ls aiss://$BUCKET_1 // FAIL "invalid backend provider "aiss""

ais bucket rm ais://$BUCKET_1 // IGNORE
