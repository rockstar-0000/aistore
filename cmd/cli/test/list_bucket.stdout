NAME			 SIZE
tmp/shard-0.tar		 2.50KiB
tmp/shard-1.tar		 2.50KiB
tmp/shard-2.tar		 2.50KiB
tmp/shard-3.tar		 2.50KiB
tmp/shard-4.tar		 2.50KiB
NAME			 SIZE
tmp/shard-0.tar		 2.50KiB
tmp/shard-1.tar		 2.50KiB
tmp/shard-2.tar		 2.50KiB
tmp/shard-3.tar		 2.50KiB
tmp/shard-4.tar		 2.50KiB
NAME			 SIZE
tmp/shard-0.tar		 2.50KiB
tmp/shard-1.tar		 2.50KiB
tmp/shard-2.tar		 2.50KiB
tmp/shard-3.tar		 2.50KiB
tmp/shard-4.tar		 2.50KiB
NAME			 SIZE	 VERSION
tmp/shard-0.tar		 2.50KiB	 1
tmp/shard-1.tar		 2.50KiB	 1
tmp/shard-2.tar		 2.50KiB	 1
tmp/shard-3.tar		 2.50KiB	 1
tmp/shard-4.tar		 2.50KiB	 1
NAME			 SIZE	 VERSION
tmp/shard-0.tar		 2.50KiB	 1
tmp/shard-1.tar		 2.50KiB	 1
tmp/shard-2.tar		 2.50KiB	 1
tmp/shard-3.tar		 2.50KiB	 1
tmp/shard-4.tar		 2.50KiB	 1
NAME	 SIZE		 CHECKSUM	 ATIME	 VERSION	COPIES CUSTOM	LOCATION       STATUS
^tmp/shard-0.tar	 2.50KiB	 (.*)	 .{16}	 1	1	-	t(.*):mp(.*)   ok$
^tmp/shard-1.tar	 2.50KiB	 (.*)	 .{16}	 1	1	-	t(.*):mp(.*)   ok$
^tmp/shard-2.tar	 2.50KiB	 (.*)	 .{16}	 1	1	-	t(.*):mp(.*)   ok$
^tmp/shard-3.tar	 2.50KiB	 (.*)	 .{16}	 1	1	-	t(.*):mp(.*)   ok$
^tmp/shard-4.tar	 2.50KiB	 (.*)	 .{16}	 1	1	-	t(.*):mp(.*)   ok$
NAME			 SIZE
tmp/shard-0.tar		 2.50KiB
tmp/shard-1.tar		 2.50KiB
tmp/shard-2.tar		 2.50KiB
tmp/shard-3.tar		 2.50KiB
tmp/shard-4.tar		 2.50KiB
